skabase/SKAMaster/SKAMaster.py

```diff
@@ -178,10 +178,11 @@ def isCapabilityAchievable(self, argin):
         # PROTECTED REGION ID(SKAMaster.isCapabilityAchievable) ENABLED START #
         """
         Checks of provided capabilities can be achieved by the resource(s).
-        :param argin: DevVarLongStringArray. An array consisting pair of DevLong and
-        DevString. A single entry consists of\n
-            [nrInstances]: DevLong. Number of instances of the capability
-            [Capability types]: DevString. Type of capability
+
+        :parameter argin: DevVarLongStringArray. An array consisting pair of
```

Comment:
In the other files modified in this PR, the docstrings are using `:param`, not `:parameter`.  Those other files also have an extra blank line before describing the parameter.

What standard are we following for docstrings?

Reply:
Modified the documentation. 
We are using Sphinx style docstrings in general. Although we know the docstrings can be improved further, both in terms of content and format. We are doing it steadily.